pygotham/frontend/templates/home/index.html

```diff
@@ -1,12 +1,17 @@
 {% extends 'layouts/base.html' %}
 
+{% block extra_head %}
+<link href='https://fonts.googleapis.com/css?family=Aldrich' rel='stylesheet' type='text/css'>
```

Comment:
Is there a reason this is here and not [in the CSS](https://github.com/PyGotham/pygotham/blob/386a1131db75e2b8c3f4fb07b02d2ef4595a39d1/pygotham/frontend/static/scss/app.scss#L4)?